inst/extdata/packages/bamm.yaml

```
@@ -0,0 +1,51 @@
+name : bamm
+title: "Species Distribution Models as a Function of Biotic, Abiotic and
+       Movement Factors (BAM)"
+version: 2.0.4
+author: "Luis Osorio-Olvera, Jorge Soberón"
+maintainer: "Luis Osorio-Olvera <luismurao@gmail.com>"
+cran: yes
+repository: "https://github.com/luismurao/bamm"
+description: "Species Distribution Modeling (SDM) is a practical methodology
+ that aims to estimate the area of distribution of a species. However,
+ most of the work has focused on estimating static expressions of the
+ correlation between environmental variables. The outputs of
+ correlative species distribution models can be interpreted
+ as maps of the suitable environment for a species but not generally as maps
+ of its actual distribution.
+ Soberón and Peterson (2005) <doi:10.17161/bi.v2i0.4> presented
+ the BAM scheme, a heuristic framework that states that the occupied area of
+ a species occurs on sites that have been accessible through dispersal (M) and
+ have both favorable biotic (B) and abiotic conditions (A).
+ The 'bamm' package implements classes and functions to operate on each element
+ of the BAM and by using a cellular automata model where the occupied area
+ of a species at time t is estimated by the multiplication of three
+ binary matrices: one matrix represents movements (M), another
+ abiotic -niche- tolerances (A), and a third, biotic interactions (B).
+ The theoretical background of the package can be found in
+ Soberón and Osorio-Olvera (2022) <arXiv:2212.06308>."
+occ_acquisition: no
+occ_cleaning: no
+data_integration: no
+env_collinearity: no
+env_process: no
+bias: no
+study_region: no
+backg_sample: no
+data_partitioning: no
+mod_fit: no
+mod_tuning: no
+mod_ensemble: no
+mod_stack: yes
+mod_evaluate: no
+mod_multispecies: yes
```

```suggestion
mod_multispecies: no
```
In my understanding from the explanation, it seems to me more that `bamm` is doing post-processing analysis on distributional models for several species. I think `mod_multispecies` refers more to whether the package itself *generates* models for multiple species. @sylvainschmitt, is that what you meant?

---

To summarize, what we wrote with @jamiemkass & @adamlilith is:

> mod_multispecies: Function(s) that model multiple species at once to estimate community composition / diversity (i.e., not a “stacked model; yes/no)

And what @luismurao described is:

> - bamm::community_sim. Estimate community dynamics using the bamm framework
> - bamm::diversity_range_analysis. Diversity range analysis and estimation of biodiversity indices (alpha, dispersion field, gamma) related to diversity-range plots.

I feel this is more about post-processing too, but I'm a bit unsure. I should not decide alone the rules. So @luismurao you can explain further why you think it should be included in multispecies modelling. And @hannahlowens , @jamiemkass  and @adamlilith I'm interested by what your minds on the question.

---

I guess it depends on whether or not `bamm` makes the models that generate the predictions which are used to estimate diversity indices or not. If the answer is no, I would not call it a "multispecies SDM" because it is using SDM results (post-processing, as Sylvain said).